yarn.lock

```diff
@@ -3855,6 +3879,16 @@
     "@ledgerhq/logs" "^6.12.0"
     events "^3.3.0"
 
+"@ledgerhq/hw-transport@^6.30.6":
```

Comment:
We already have `@ledgerhq/hw-transport@^6.30.5`, Could we remove that and use `@ledgerhq/hw-transport@^6.30.6` instead?

Reply:
Same thing for `@ledgerhq/errors` and `@ledgerhq/devices`

Reply:
fixed by upgrading `@ledgerhq/hw-app-eth`